### ISSUE TITLE
After latest update no more FormsPlot.mouseCoordinates method available

### ISSUE THREAD

**Issue by bukkideme** [swharden/ScottPlot#235]
Hello!

The latest nuget update broke one of my code part:
```
private void formsPlot1_MouseClicked(object sender, MouseEventArgs e)
        {
            toolStripTextBox1.Text = $"X: {Math.Round(formsPlot1.mouseCoordinates.X)}, Y: {Math.Round(formsPlot1.mouseCoordinates.Y)}";            
        }
```

> Error	CS1061	'FormsPlot' does not contain a definition for 'mouseCoordinates' and no accessible extension method 'mouseCoordinates' accepting a first argument of type 'FormsPlot' could be found (are you missing a using directive or an assembly reference?)

**Comment by swharden** [comment]
@bukkideme thanks for noting this!

The last update refactored how mouse tracking is accomplished and removal of this line is what broke your code:
https://github.com/swharden/ScottPlot/blob/d12189309d7ca40ea9881e830e3af3ab2e7fc43b/src/ScottPlot.WinForms/FormsPlot.cs#L96

I'll add this back in. In the mean time, you could get your code working by changing your line to:
```cs
var coord = formsPlot1.plt.CoordinateFromPixel(e.Location);
toolStripTextBox1.Text = $"X: {Math.Round(coord.X)}, Y: {Math.Round(coord.Y)}";
```

**Comment by swharden** [comment]
Just released ScottPlot 4.0.17 on NuGet. It should be live shortly! Let me know if this doesn't do the trick. Thanks again @bukkideme

**Comment by bukkideme** [comment]
Thanks it works again!
